wasm2c/examples/fac/fac.c

```diff
@@ -66,11 +66,12 @@ static inline bool func_types_eq(const wasm_rt_func_type_t a,
 #define MEMCHECK(mem, a, t) RANGE_CHECK(mem, a, sizeof(t))
 #endif
 
-#ifdef __GNUC__
-#define wasm_asm __asm__
+#if defined(__GNUC__) && (defined(__x86_64__) || defined(__aarch64__))
```

Comment:
Should enable this by default and just allow it to be turned off selectively.. ?    Maybe we could #error out to first folks on MIPS to opt out (e.g. `#error no support for strict trapping on this architecture.. you can opt out using -DWASM2C_DISABLED_STRICT_TRAPPING` )